lib/resources/r4/charge_item_modify.yaml

```diff
@@ -0,0 +1,417 @@
+        }
+      }
+    note: >
+      <ul>
+        <li>Must be an Encounter reference</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.context
+
+  - name: occurrenceDateTime
+    required: 'No'
+    type: dateTime
+    description: Date/time when the charged service was applied.
+    action: modify
+    example: |
+      {
+        "occurrenceDateTime": "2021-01-08T10:38:00Z"
+      }
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.occurrenceDateTime
+
+  - name: quantity
+    required: 'No'
+    type: Quantity
+    description: Quantity of which the charge item has been serviced.
+    action: modify
+    example: |
+      {
+        "quantity": {
+          "value": 31
+        }
+      }
+    note: >
+      <ul>
+        <li>The value must be in the form of `integer`</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.quantity
+
+  - name: code
+    required: 'Yes'
+    type: CodeableConcept
+    description: The primary billing code that identifies the charge.
+    action: modify
+    example: |
+      {
+        "code": {
+          "coding": [
+            {
+              "extension": [
+                {
+                  "valueCoding": {
+                    "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/codeSet/14002",
+                    "code": "667687"
+                  },
+                  "url": "https://fhir-ehr.cerner.com/r4/StructureDefinition/bill-code-schedule"
+                }
+              ],
+              "system": "https://fhir.cerner.com/ec2458f2-1e24-41c8-b71b-0e701af7583d/CodeSystem/BillCodes-CDM_SCHED",
+              "code": "64897"
+            }
+          ]
+        }
+      }
+    note: >
+      <ul>
+        <li>The coding field must contain only one coding from the CDM_SCHED, CPT, HCPCS, or PROCCODE proprietary Bill Codes Types systems.</li>
+        <li>Tiering logic from highest to lowest: CDM_SCHED, CPT, HCPCS, or PROCCODE</li>
+      </ul>
+    url: https://hl7.org/fhir/r4/chargeitem-definitions.html#ChargeItem.code
+
+  - name: performer
+    required: 'No'
+    type: List of BackboneElement
+    description: Indicates who performed or participated in the charge.
+    action: modify
+    example: |
+      {
+        "performer": [
+          {
+            "function": {
+              "coding": [
+                {
+                  "system": "http://terminology.hl7.org/CodeSystem/v2-0912",
+                  "code": "OP",
+                  "display": "Ordering Provider"
+                }
+              ],
+              "text": "Ordering Provider"
+            },
+            "actor": {
+              "reference": "Practitioner/11817978"
+            }
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>Need notes around supported system and values for function</li>
+        <li>Both actor and function are required</li>
```

Comment:
nit: <code>Must contain both the `actor` and `function` subfields.</code>

Reply:
updated here https://github.com/cerner/fhir.cerner.com/pull/780/commits/4595890bdc6f918a94d93cd54ca164dab864e8e0#diff-3c565d9c41e2bb3d417efe63a774683e72b07cd38f2ff54a695c695999f29938R161